packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -16,17 +17,27 @@ import { useBaseTreeItem_unstable } from '../BaseTreeItem/index';
  */
 export const useTreeItem_unstable = (props: TreeItemProps, ref: React.Ref<TreeItemElement>): TreeItemState => {
   const treeItemState = useBaseTreeItem_unstable(props, ref);
-  const { expandIcon, iconBefore, iconAfter, actionIcon } = props;
+  const { expandIcon, iconBefore, iconAfter, actions, badges } = props;
   const { dir } = useFluent_unstable();
   const expandIconRotation = treeItemState.open ? 90 : dir !== 'rtl' ? 0 : 180;
+
+  // stop the propagation of a click from actions to ensure it doesn't open the treeitem
+  const handleActionsClick = useEventCallback((event: React.MouseEvent<HTMLElement>) => {
```

Comment:
should this stop propagation or prevent default? @bsunderhus  I know that you've generally preferred default preventing

Reply:
both would work in this scenario, but yeah... I'm mostly in favor of preventing the default to indicate we're opting out of a behaviour. I'll update this  😁